src/actions/login.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios'
+import {
+    urlLogin,
+    urlRegister
+} from '../urls'
+import {
+    LOGIN,
+    REGISTER,
+    LOGIN_ERRORS,
+    REGISTER_ERRORS
+} from './types'
+
+export const postLogin = (data, callback) => async dispatch => {
+    try {
+        const config = {
+            headers: {
+                'content-type': 'application/json',
+            }
+        }
+        const res = await axios.post(urlLogin(), data, config);
+        dispatch({
+            type: LOGIN,
+            payload: res.data
+        });
+        localStorage.setItem('token', res.data.access)
+        callback()
+    }
+    catch (err) {
+        dispatch({
+            type: LOGIN_ERRORS,
+            payload: err.response.data
+        });
+        callback()
+    }
+};
+
+export const postRegister = (data, callback) => async dispatch => {
+    try {
+        const config = {
+            headers: {
+                'content-type': 'application/json',
+            }
+        }
+        const res = await axios.post(urlRegister(), data, config);
+        dispatch({
+            type: REGISTER,
+            payload: res.data
+        });
+        callback()
+    }
+    catch (err) {
+        dispatch({
+            type: REGISTER_ERRORS,
+            payload: err.response.data
+        });
+        callback()
+    }
+};
```

Comment:
Add new line

Reply:
Merging and rebasing will work, actually these changes are coming from previous commits which are related to the previous PRs. So these new line problems are coming from those commits.